src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -84,21 +84,6 @@ public async Task BestLocations_EmptyLocationQueryString_ReturnsBadRequest(strin
         Assert.That(result.StatusCode, Is.EqualTo(HttpStatusCode.BadRequest));
```

Comment:
In VS Code, the CarbonAware.WebApi.IntegrationTests and CarbonAware.WebApiUnitTests are missing references, or the paths are incorrect in the csproj file and are failing.

Reply:
When running outside of Visual Studio ™️ I often need to run `dotnet clean` before `dotnet build` and/or `dotnet test` otherwise old artifacts will confuse VSCode or prevent building altogether.  Especially true when changing branches.  Let me know if you are still seeing those errors after running `dotnet clean` first.

Reply:
I run dotnet clean before I run dotnet build.  It appears that the "Program" object needs a Visual Studio package that may not need to be included in the using statements in VS, but that is a guess from what I saw using the "Quick Fix" option.